admin/modules/user/users.php

```diff
@@ -3959,6 +3959,7 @@ function output_custom_profile_fields($fields, $values, &$form_container, &$form
 		switch($type)
 		{
 			case "multiselect":
+				$selected_options = array();
```

Comment:
Just a styling tip, can you add a newline before this? Thank you, keep it up!

Reply:
Since when was a newline after the case statement part of the standards? The example given in the [development standards](https://docs.mybb.com/1.8/development/standards/) doesn't have an extra newline.

Reply:
Yep my bad, already asked @fluxtion to remove it below.

Reply:
Ah yes, seen now! 🙂